code/src/App.js

```diff
@@ -1,9 +1,69 @@
-import React from 'react'
+import React, { useState, useEffect } from 'react'
+import { ThoughtsList } from './ThoughtsList'
+import { ThoughtInput } from './ThoughtInput'
+
+const THOUGHTS_URL = 'https://technigo-thoughts.herokuapp.com/'
 
 export const App = () => {
+  const [thought, setThought] = useState('')
+  const [thoughts, setThoughts] = useState([])
+
+  const handleSubmit = event => {
+    event.preventDefault()
+    setThought('')
+
+    fetch(THOUGHTS_URL,
+      {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json'
+        },
+        body: JSON.stringify({ message: thought })
+      }
+    ).then(res => res.json())
+      .then(newThought => {
+        setThoughts([newThought, ...thoughts])
+      })
+  }
+
+  useEffect(() => {
+    fetch(THOUGHTS_URL)
+      .then((res) => {
+        return res.json()
+      })
+      .then(data => {
+        setThoughts(data)
+      })
+  }, [])
+
+  const sendHeart = (id) => {
```

Comment:
In case you didn't know - you don't need `()` if you are passing only one argument
```suggestion
  const sendHeart = id => {
```